components/FilterBar/components/Content/index.jsx

```diff
@@ -1,6 +1,8 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import { compareObjects } from '@shopgate/pwa-common/helpers/redux';
+/** @type {{currency: string}} */
```

Comment:
Please remove this line. We don't have this anywhere. No reason to start it now.

Reply:
+